docs/proposals/fluentd-secure-forward.md

```diff
@@ -0,0 +1,35 @@
+# Allow Fluentd to secure forward to other Fluentd/Logstash (over ES-Copy)
```

Comment:
We should mention in this document something about changes to the deployer for cases where only fluentd will be deployed to support an external aggregation (e.g. Splunk).  Are we going to have single flag?  Maybe there is a 'history' captured in the configmap which might allow something like diagnostics to only evaluate the relevant components?